website/src/components/wordpress.tsx

```diff
@@ -81,7 +81,8 @@ const parseOptions: HTMLReactParserOptions = {
   replace(node) {
     if ("data" in node) {
       const referenceEmbedStyle = /\[(\w*)\]/ // [search | glossary]
-      const wordEmbedStyle = /\[(\w*):([0-9]*)-?([0-9]*)?:?(audio)?(join)?(#)?(\w*)?\]/ // [DocName:Start(-OptionalEnd):?(audio?)(join?)#?OptionalChapterSlug?]
+      const wordEmbedStyle =
+        /\[(\w*):([0-9]*)-?([0-9]*)?:?(audio)?(join)?(#)?(\w*)?\]/ // [DocName:Start(-OptionalEnd):?(audio?)(join?)#?OptionalChapterSlug?]
```

Comment:
@GracefulLemming we may be in a formatting battle - my `dev-check` changed this, but the last edit was you and the commit message was `prettier`

Reply:
try merging main in– running `prettier -w src` and `prettier -w --config package.json src` yields two different outputs. All of our checks use the one with `--config` specified so thats likely what I ran